apps/privacy-panel/js/ala/blur_slider.js

```diff
@@ -117,18 +120,18 @@ function() {
    */
   BlurSlider.getLabel = function(value) {
     switch(parseInt(value)) {
-      case 1:   return '500m';
-      case 2:   return '1km';
-      case 3:   return '2km';
-      case 4:   return '5km';
-      case 5:   return '10km';
-      case 6:   return '15km';
-      case 7:   return '20km';
-      case 8:   return '50km';
-      case 9:   return '75km';
-      case 10:  return '100km';
-      case 11:  return '500km';
-      case 12:  return '1000km';
+      case 1:   return '500 '+navigator.mozL10n.get('blur-unit-m');
+      case 2:   return '1 '+navigator.mozL10n.get('blur-unit-km');
+      case 3:   return '2 '+navigator.mozL10n.get('blur-unit-km');
+      case 4:   return '5 '+navigator.mozL10n.get('blur-unit-km');
+      case 5:   return '10 '+navigator.mozL10n.get('blur-unit-km');
+      case 6:   return '15 '+navigator.mozL10n.get('blur-unit-km');
+      case 7:   return '20 '+navigator.mozL10n.get('blur-unit-km');
+      case 8:   return '50 '+navigator.mozL10n.get('blur-unit-km');
+      case 9:   return '75 '+navigator.mozL10n.get('blur-unit-km');
+      case 10:  return '100 '+navigator.mozL10n.get('blur-unit-km');
+      case 11:  return '500 '+navigator.mozL10n.get('blur-unit-km');
+      case 12:  return '1000 '+navigator.mozL10n.get('blur-unit-km');
```

Comment:
actually, I was wrong. Because you may want to localize 'km' and 'm', you should have two entities:

blur-distance-km = {{value}}km
blur-distance-m = {{value}}m

and you should select the L10nID based on the tuples value unit and pass value as an l10nArg.